audit/gdcd/README.md

```diff
@@ -121,11 +124,22 @@ projects are parsed. Depending on your machine and the amount of projects specif
 long-running program (~1-2hrs ). 
 
 ## Troubleshooting
+### Permission Issues
+```text
+(AtlasError) user is not allowed to do action [...] on [...]
+```
+1. Verify that your Atlas user has the
+   [`dbAdminAnyDatabase`](https://www.mongodb.com/docs/manual/reference/built-in-roles/#mongodb-authrole-dbAdmin)
```

Comment:
Why set up a custom role if it's using a default permission option - i.e. `dbAdminAnyDatabase`? Why not just specify that this is the required role?